pom.xml

```diff
@@ -29,15 +29,12 @@
   <properties>
     <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
```

Comment:
Please add checkstyle and rat plugins.
Also suppressions.xml and checkstyle.xml files.

Reply:
Done